cfgov/ask_cfpb/models/pages.py

```diff
@@ -523,13 +528,23 @@ class AnswerPage(CFGOVPage):
     Page type for Ask CFPB answers.
     """
     from ask_cfpb.models import Answer
-    question = RichTextField(blank=True, editable=False)
-    answer = RichTextField(blank=True, editable=False)
-    snippet = RichTextField(
-        blank=True, help_text='Optional answer intro', editable=False)
-    created_at = models.DateTimeField(auto_now_add=True)
-    updated_at = models.DateTimeField(auto_now=True)
-    publish_date = models.DateTimeField(default=timezone.now)
+    last_edited = models.DateField(
+        blank=True,
+        null=True,
+        help_text="Change the date to today if you make a significant change.")
+    question = models.TextField(blank=True)
```

Comment:
Is there a reason the `question` field becomes a `TextField` in place of a `RichTextField`?

Reply:
The question has always been a TextField in the Answer model. We didn't see a need to allow  markup in questions.